micronaut-camunda-bpm-example/build.gradle

```diff
@@ -6,23 +6,42 @@ plugins {
 group "info.novatec"
 
 micronaut {
-    runtime "netty"
+    // runtime "netty" Commented for JETTY EXAMPLE
```

Comment:
I think this can in the end be reverted - we can keep it as a reminder to test both for now